wolfi-dev/os
# perl: test2-suite got merged into test-simple

perl-test2-suite has file conflicts with perl-test-simple, as it got
merged into it upstream. In 2016 -
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=823958

- Remove perl-test2-suite
- Add perl-test2-suite provides to perl-test-simple
- Update all other reverse-dependenices
- Rebuild lcov to check that it now builds

Fixes:

```console
$ make package/lcov
...
2024/08/14 08:59:14 INFO installing perl-test-simple (1.302200-r0)
2024/08/14 08:59:14 INFO installing perl-module-pluggable (5.2-r2)
2024/08/14 08:59:14 INFO installing perl-importer (0.026-r3)
2024/08/14 08:59:14 INFO installing perl-sub-info (0.002-r2)
2024/08/14 08:59:14 INFO installing perl-scope-guard (0.21-r3)
2024/08/14 08:59:14 INFO installing perl-term-table (0.021-r0)
2024/08/14 08:59:14 INFO installing perl-test2-suite (0.000163-r1)
2024/08/14 08:59:14 ERRO ERROR: failed to build package. the build environment has been preserved:
2024/08/14 08:59:14 INFO   workspace dir: /tmp/melange-workspace-1596319423
2024/08/14 08:59:14 INFO   guest dir: /tmp/melange-guest-2396889605
2024/08/14 08:59:14 ERRO failed to build package: unable to build guest: unable to generate image: installing apk packages: installing packages: installing perl-test2-suite (ver:0.000163-r1 arch:x86_64): unable to install files for pkg perl-test2-suite: unable to install file over existing one, different contents: usr/share/perl5/vendor_perl/Test2/AsyncSubtest/Event/Attach.pm
```

No idea how any of this got built the first time around.
